Makefile

```diff
@@ -25,7 +25,7 @@ check: fmt vet lint test ## Check your code
 
 .PHONY: test
 test: # Run unit test
-	$(DOCKER_CMD) go test -race -cover ./pkg/...
+	$(DOCKER_CMD) go test -race -cover `go list ./... | grep -v github.com/openshift/cluster-api-actuator-pkg/pkg/e2e`
```

Comment:
Why don't we be explicit here? Just run the named package?

Reply:
other packages might have unit tests worth running so I am just skipping anything but e2e